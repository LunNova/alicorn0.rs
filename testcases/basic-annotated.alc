let host-string-wrap          = intrinsic "return terms.strict_value.host_string_type"    : wrapped(host-type)

let host-string          = unwrap(host-string-wrap)
