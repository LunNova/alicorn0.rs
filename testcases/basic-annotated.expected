{ { "let", "host-string-wrap", "=", "intrinsic", { elements = { "return terms.strict_value.host_string_type" }, kind = "string" }, ":", { "wrapped", "host-type" } }, { "let", "host-string", "=", { "unwrap", "host-string-wrap" } } }
